packages/vx-gradient/src/gradients/LinearGradient.tsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+
+type LinearGradientElementProps = React.SVGProps<SVGLinearGradientElement>;
+type StopElementProps = React.SVGProps<SVGStopElement>;
+
+export type LinearGradientProps = {
+  /** Unique id for the gradient. Should be unique across all page elements. */
+  id: string;
+  /** Start color of gradient. */
+  from?: StopElementProps['stopColor'];
```

Comment:
@milesj I noticed that none of these get converted to `prop-types`, is there any workaround for that? 

Most of these are just strings/numbers so I could remove the reliance on `React.SVGProps`

Reply:
It doesn't generate prop types because these require TS type information. The Babel plugin only uses the raw source for information.